public/app/features/serviceaccounts/ServiceAccountsListPage.tsx

```diff
@@ -193,7 +192,7 @@ export const ServiceAccountsListPageUnconnected = ({
             onRemove={onMigrationInfoClose}
           ></Alert>
         )}
-        {!config.featureToggles.topnav && (
+        <Page.OldNavOnly>
```

Comment:
Replacing this old feature toggle check with the new Page.OldNavOnly component that helps make this type of logic more reusable / readble